Detectors/TPC/reconstruction/include/TPCReconstruction/TPCCATracking.h

```diff
@@ -0,0 +1,52 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TPCCATracking.h
+/// \brief Wrapper class for TPC CA Tracker algorithm
+/// \author David Rohr
+#ifndef ALICEO2_TPC_TPCCATRACKING_H_
+#define ALICEO2_TPC_TPCCATRACKING_H_
+
+#include <memory>
+#include <vector>
+class TClonesArray;
+class AliHLTTPCCAO2Interface;
+class AliHLTTPCCAClusterData;
+
+namespace o2
+{
+namespace TPC
+{
+
+class TrackTPC;
+
+class TPCCATracking
+{
+public:
+  TPCCATracking();
+  ~TPCCATracking();
+
+  int initialize(const char* options = nullptr);
+  void deinitialize();
+
+  int runTracking(const TClonesArray* inputClusters, std::vector<TrackTPC>* outputTracks);
+
+private:
+  std::unique_ptr<AliHLTTPCCAO2Interface> mTrackingCAO2Interface;
+  std::unique_ptr<AliHLTTPCCAClusterData[]> mClusterData_UPTR;
+  AliHLTTPCCAClusterData* mClusterData;
+
+  TPCCATracking(const TPCCATracking&);            // Disable copy
+  TPCCATracking& operator=(const TPCCATracking&); // Disable assignment
```

Comment:
with C++11 we can be more explicit and just say `TPCCATracking(const ...) = delete;`